src/h2olog/h2olog.cc

```diff
@@ -264,7 +314,13 @@ int main(int argc, char **argv)
     }
 
     ebpf::BPF *bpf = new ebpf::BPF();
-    std::vector<ebpf::USDT> probes = tracer->init_usdt_probes(h2o_pid);
+    std::vector<ebpf::USDT> probes;
+    for (const auto &probe_def : tracer->usdt_probes()) {
+        if (event_type_filters.empty() || std::find(event_type_filters.cbegin(), event_type_filters.cend(),
```

Comment:
I've chosen `std::vector` instead of `std::ordered_map` to keep its order in debug print (`-dd`). It doesn't affect tracing performance, anyway.